src/index.js

```diff
@@ -80,6 +81,12 @@ export default class MediaQuery extends React.Component {
       matches: this._mql.matches
     })
   }
+  
+  componentDidUpdate(_, prevState) {
```

Comment:
componentWillUpdate ? I think you'd want this to fire before the render happens, right?